### PR TITLE
update pdir extensions

### PR DESCRIPTION
**Pull request by seibtph**

### CODE REVIEW

File: docs/manual/extensions/maklermodul.en.md [hunk 1]
```diff
@@ -14,3 +14,23 @@ The broker module for estate agents is a paid extension for the Contao CMS.
 ![Broker list view in frontend](/de/extensions/images/de/maklermodul-bundle-frontend.png)
 
 Detailed instructions can be found in the [manual](https://docs.pdir.de/#/maklermodul/index).
+
+### Contao Themes with layout templates for the Maklermodul
+
+This contao themes includes layout templates for the Maklermodul.
```
Comment (fritzmg) [inline docs/manual/extensions/maklermodul.en.md]:
```suggestion
These contao themes include layout templates for the Maklermodul.
```

File: docs/manual/extensions/maklermodul.en.md [hunk 2]
```diff
@@ -14,3 +14,23 @@ The broker module for estate agents is a paid extension for the Contao CMS.
...
+
+#### MATE Theme
+
+**Demo:** [mate.pdir.de/immobilien.html](https://mate.pdir.de/immobilien.html)  
+**Docs:** [docs.contao-themes.net](https://docs.contao-themes.net/#/mate_theme/maklermodul)  
+**Install for free or order a license:** [contao-themes.net/theme-detail/mate.html](https://contao-themes.net/theme-detail/mate.html)  
+
+![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![Immobilienliste mit Buttons im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![Immobilien Exposé im MATE Theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
```
Comment (fritzmg) [inline docs/manual/extensions/maklermodul.en.md]:
```suggestion
![property list with mit select lists within the MATE theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![property list with buttons within the MATE theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![property description within the MATE theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
```

File: docs/manual/extensions/maklermodul.en.md [hunk 3]
```diff
@@ -14,3 +14,23 @@ The broker module for estate agents is a paid extension for the Contao CMS.
...
+
+#### ODD Theme
+
+**Demo:** [odd.contao-themes.net/immobilien.html](https://odd.contao-themes.net/immobilien.html)  
+**Docs:** [docs.contao-themes.net](https://docs.contao-themes.net/#/odd_theme/odd-maklermodul)  
+**Install for free or order a license:** [contao-themes.net/theme-detail/odd.html](https://contao-themes.net/theme-detail/odd.html)  
+
+![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_odd_liste_selects.png ':size=150') ![Immobilienliste mit Buttons im MATE Theme](/de/extensions/images/de/maklermodul_odd_liste_buttons.png ':size=150') ![Immobilien Exposé im MATE Theme](/de/extensions/images/de/maklermodul_odd_details.png ':size=150')
```
Comment (fritzmg) [inline docs/manual/extensions/maklermodul.en.md]:
```suggestion
![property list with select lists within the ODD theme](/de/extensions/images/de/maklermodul_odd_liste_selects.png ':size=150') ![property list with buttons within the ODD theme](/de/extensions/images/de/maklermodul_odd_liste_buttons.png ':size=150') ![property description within the ODD theme](/de/extensions/images/de/maklermodul_odd_details.png ':size=150')
```

**Review by fritzmg — CHANGES_REQUESTED** [review]
I haven't looked at the images yet, but may be you can provide English versions too?

**Review by fritzmg — DISMISSED** [review]

File: docs/manual/extensions/maklermodul.en.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ These contao themes include layout templates for the Maklermodul.
 **Docs:** [docs.contao-themes.net](https://docs.contao-themes.net/#/mate_theme/maklermodul)  
 **Install for free or order a license:** [contao-themes.net/theme-detail/mate.html](https://contao-themes.net/theme-detail/mate.html)  
 
-![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![Immobilienliste mit Buttons im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![Immobilien Exposé im MATE Theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
+![property list with mit select lists within the MATE theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![property list with buttons within the MATE theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![property description within the MATE theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
```
Comment (fiedsch) [inline docs/manual/extensions/maklermodul.en.md]: remove "mit"

**Review by fritzmg — CHANGES_REQUESTED** [review]
Btw. please use line breaks after 80 characters, or 120 characters at the most.

**Comment by seibtph** [comment]
> Btw. please use line breaks after 80 characters, or 120 characters at the most.

Thanks, I have changed this.

File: docs/manual/extensions/animated-timeline.de.md [hunk 5]
```diff
@@ -8,9 +8,12 @@ url: "erweiterungen/animated-timeline"
 
 _von [pdir GmbH](https://www.pdir.de)_
 
-Das Animated Timeline Bundle ist eine jQuery-Erweiterung für Contao 4 auf Basis von jQuery Roadmap, mit der man Inhalte in einem dynamischen Zeitstrahl mit fadeIn-Animation darstellen kann.
+Das Animated Timeline Bundle ist eine jQuery-Erweiterung für Contao 4 auf Basis von jQuery Roadmap, mit der man Inhalte
```
Comment (netzarbeiter) [inline docs/manual/extensions/animated-timeline.de.md]: Can you please replace «jQuery Roadmap» with «jQuery-Roadmap».

File: docs/manual/extensions/maklermodul.de.md [hunk 6]
```diff
@@ -26,12 +26,16 @@ Diese Themes liefern bereits Layoutvorlagen (CSS und Templates) für das Maklerm
 **Dokumentation:** [docs.contao-themes.net](https://docs.contao-themes.net/#/mate_theme/maklermodul)  
 **Kostenlos installieren oder Lizenz bestellen:** [contao-themes.net/theme-detail/mate.html](https://contao-themes.net/theme-detail/mate.html)  
 
-![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![Immobilienliste mit Buttons im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![Immobilien Exposé im MATE Theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
+![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150')
```
Comment (netzarbeiter) [inline docs/manual/extensions/maklermodul.de.md]: Can you please remove « ':size=150'» in every image tag. If it is really necessary to resize the image, you will find the correct syntax in the [documentation](https://themes.gohugo.io//theme/hugo-theme-learn/en/cont/markdown/#resizing-image).

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you @seibtph, can you please make the changes.

**Review by netzarbeiter — APPROVED** [review]

**Review by fritzmg — APPROVED** [review]

**Pull request merged**